nginx/default.toml

```diff
@@ -19,6 +19,12 @@ worker_processes = 4
 # worker_connections: Connections per Worker Process.  Default = 1024
 worker_connections = 1024
 
+[[include.additional_configuration_file]]
+config_file = "mime.types"
+
+[[include.additional_configuration_file]]
```

Comment:
I'm curious about the toml syntax pattern here - this appears verbose to add another config_file inclusion - isn't there a better way to express arrays of items in toml?